src/GTranslate/LanguageDictionary.cs

```diff
@@ -142,153 +142,167 @@ public bool TryGetLanguage(string code, [MaybeNullWhen(false)] out Language lang
+        ["mai"] = new("Maithili", "Macushi", "mai", "mai", TranslationServices.Google),
         ["mg"] = new("Malagasy", "Malagasy", "mg", "mlg"),
-        ["ms"] = new("Malay", "Melayu", "ms", "msa"),
-        ["ml"] = new("Malayalam", "മലയാളം", "ml", "mal"),
-        ["mt"] = new("Maltese", "Malti", "mt", "mlt"),
+        ["mhr"] = new("Eastern Mari", "олык марий", "mhr", "mhr", TranslationServices.Yandex),
         ["mi"] = new("Maori", "Te Reo Māori", "mi", "mri"),
-        ["mr"] = new("Marathi", "मराठी", "mr", "mar"),
+        ["mk"] = new("Macedonian", "Македонски", "mk", "mkd"),
+        ["ml"] = new("Malayalam", "മലയാളം", "ml", "mal"),
         ["mn"] = new("Mongolian", "Монгол хэл", "mn", "mon"),
+        ["mn-Mong"] = new("Mongolian (Traditional)", "ᠮᠣᠩᠭᠣᠯ ᠬᠡᠯᠡ", "mn-Mong", "mn-Mong", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["mr"] = new("Marathi", "मराठी", "mr", "mar"),
+        ["mrj"] = new("Western Mari", "Мары йӹлмӹ", "mrj", "mrj", TranslationServices.Yandex),
+        ["ms"] = new("Malay", "Melayu", "ms", "msa"),
+        ["mt"] = new("Maltese", "Malti", "mt", "mlt"),
+        ["mww"] = new("Hmong Daw", "Hmong Daw", "mww", "mww", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["my"] = new("Burmese", "မြန်မာ", "my", "mya"),
         ["ne"] = new("Nepali", "नेपाली", "ne", "nep"),
+        ["nl"] = new("Dutch", "Nederlands", "nl", "nld"),
         ["no"] = new("Norwegian", "Norsk", "no", "nor"),
+        ["nso"] = new("Sepedi", "Sesotho sa Leboa", "nso", "nso", TranslationServices.Google),
+        ["ny"] = new("Chichewa", "Nyanja", "ny", "nya", TranslationServices.Google),
+        ["om"] = new("Oromo", "Afaan Oromoo [ʌfɑːn ɔrɔmoː] Oromiff", "om", "orm", TranslationServices.Google),
         ["or"] = new("Odia", "ଓଡ଼ିଆ", "or", "ori", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
-        ["ps"] = new("Pashto", "پښتو", "ps", "pus", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
-        ["fa"] = new("Persian", "فارسی", "fa", "fas"),
+        ["otq"] = new("Querétaro Otomi", "Hñähñu", "otq", "otq", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["pa"] = new("Punjabi", "ਪੰਜਾਬੀ", "pa", "pan"),
+        ["pap"] = new("Papiamento", "Papiamento", "pap", "pap", TranslationServices.Yandex),
         ["pl"] = new("Polish", "Polski", "pl", "pol"),
+        ["prs"] = new("Dari", "دری", "prs", "prs", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["ps"] = new("Pashto", "پښتو", "ps", "pus", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
         ["pt"] = new("Portuguese", "Português", "pt", "por"),
-        ["pa"] = new("Punjabi", "ਪੰਜਾਬੀ", "pa", "pan"),
+        ["pt-BR"] = new("Portuguese (Brazil)", "Português (Brazil)", "pt-BR", "pt-BR", TranslationServices.Google),
```

Comment:
As mentioned in the comments, The APIs don't treat `pt` and `pt-BR` as separate languages, but Microsoft Azure actually does with `pt-PT`.
```suggestion
        ["pt-PT"] = new("Portuguese (Portugal)", "Português (Portugal)", "pt-PT", "pt-PT", TranslationServices.Bing | TranslationServices.Microsoft),
```

Reply:
> As mentioned in the comments, The APIs don't treat pt and pt-BR as separate languages, but Microsoft Azure actually does with pt-PT.

I'm sorry, but I still don't understand this.  I mention my confusion on this in previous post, but I didn't get a clear answer.

* It sounds like you're saying the API ignores what's after the dash.  But if that was the case, zh-CN and zh-TW would return the same results.  But they don't.  I get two separate results.  Exactly how does the API treat pt and pt-BR the same?
* When Microsoft Azure treats it as separate languages, does it produce different translation results for pt vs pt-PT?

Can you please provide more details?
Thanks

Reply:
> It sounds like you're saying the API ignores what's after the dash.

No, the APIs have a list of languages that can be used to translate. In Google Translate you can either see it in their official documentation and through the source code of translate.google.com.
The official documentation displays 109 supported languages and in translate.google.com you can see 133 supported languages. Those 2 sites display the 2 variants of chinese `zh-CN` and `zh-TW` but there's only one language code for Portuguese (`pt`).

[Google Translate](https://translate.google.com/):
![image](https://user-images.githubusercontent.com/39978852/188332029-816e4813-10e9-4f62-aa17-05c9ee1c6d8b.png)

[Google Cloud Translation docs](https://cloud.google.com/translate/docs/languages):
![image](https://user-images.githubusercontent.com/39978852/188332055-527fdae2-3020-435c-8bbf-482709928230.png)

This is different in Bing/Microsoft Azure. Both Bing Translator and Microsoft Azure Translator display separate language codes for the two Portuguese variants.

[Bing Translator](https://www.bing.com/translator):
![image](https://user-images.githubusercontent.com/39978852/188332152-161a0ea6-cd11-4bed-a86e-d00cf5e5931f.png)

[Microsoft Azure Translator docs](https://docs.microsoft.com/en-us/azure/cognitive-services/translator/language-support):
![image](https://user-images.githubusercontent.com/39978852/188332169-cd61ba6e-81f4-4a72-ac93-dbe28d496c01.png)

Reply:
I don't understand why Microsoft would default pt to Portuguese (Brazil).  Portuguese originated in Portugal, so that should be the default Portuguese for pt.

I've reverted it back.

Done.